src/service/poc_mobile.proto

```diff
@@ -7,6 +7,81 @@ import "mapper.proto";
 import "hex_boosting.proto";
 import "service_provider.proto";
 
+message hex_usage_counts_req_v1 {
+  // H3 resolution 10 tile
+  uint64 hex = 1;
+  // Number of unique service provider subscribers active in the hex
+  // during the epoch ( inclusive of discovery mapping enabled subscribers )
+  uint64 service_provider_subscriber_count = 2;
+  // Number of unique discovery mapping enabled subscribers active
+  // in the hex during the epoch
+  uint64 disco_mapping_count = 3;
+  // Number of unique offload users active in the hex during the epoch
+  uint64 offload_count = 4;
+  // Unix timestamp in milliseconds of the epoch start
+  uint64 epoch_start_timestamp = 5;
+  // Unix timestamp in milliseconds of the epoch end
+  uint64 epoch_end_timestamp = 6;
+  // Unix timestamp in milliseconds of when the report was generated
+  uint64 timestamp = 7;
+  // Carrier pubkey from verification mapping service
+  bytes carrier_mapping_key = 8;
+  // Signed payload of the verification mapping service
+  bytes signature = 9;
+}
+
+message hex_usage_counts_res_v1 { string id = 1; }
+
+message hex_usage_counts_ingest_report_v1 {
+  // Timestamp in milliseconds since unix epoch
+  uint64 received_timestamp = 1;
+  // the verified report
+  hex_usage_counts_req_v1 report = 2;
+}
+
+message radio_usage_counts_req_v1 {
+  // The onchain address of the hotspot
+  bytes hotspot_pubkey = 1;
+  // the cbsd id of the radio if it is a cbrs radio otherwise empty
+  string cbsd_id = 2;
```

Comment:
For scenario with multiple CBRS radios belonging to the same hotspot... will there exist multiple `radio_usage_counts..` events with the same `hotspot_pubkey` ?

Reply:
that is correct